{"repository":"https://github.com/adphorus/zenhub-charts","dockerfilePath":"Dockerfile","startTime":1672670049925,"clone":{"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/adphorus/zenhub-charts'...\n","error":null,"commit":"bb2a75a553e551d3c79999df7ad2e2e17e931a1d"},"originalBuild":{"startTimestamp":1672670051082,"endTimestamp":1672670080346,"error":{"code":1,"killed":false,"signal":null,"cmd":"docker build --no-cache --force-rm -t zenhub-charts:latest -f /tmp/dinghy-analysis/adphorus/zenhub-charts/Dockerfile ."},"stdout":"Sending build context to Docker daemon  885.8kB\r\r\nStep 1/10 : FROM python:3.6\n ---> 54260638d07c\nStep 2/10 : RUN mkdir /code\n ---> Running in 5f27afc1b753\nRemoving intermediate container 5f27afc1b753\n ---> a3b0cd44028b\nStep 3/10 : WORKDIR /code\n ---> Running in ab66f47d6f67\nRemoving intermediate container ab66f47d6f67\n ---> 08835a8bbd58\nStep 4/10 : ADD requirements/base.txt /code/\n ---> 9a8d9fbc63f6\nStep 5/10 : RUN pip install -r base.txt\n ---> Running in 570557883bf9\nCollecting backoff==1.4.3\n  Downloading backoff-1.4.3.tar.gz (9.9 kB)\nCollecting Django==1.11.4\n  Downloading Django-1.11.4-py2.py3-none-any.whl (6.9 MB)\nCollecting gunicorn==19.7.1\n  Downloading gunicorn-19.7.1-py2.py3-none-any.whl (111 kB)\nCollecting psycopg2==2.7.3\n  Downloading psycopg2-2.7.3-cp36-cp36m-manylinux1_x86_64.whl (2.7 MB)\nCollecting python-dateutil==2.6.1\n  Downloading python_dateutil-2.6.1-py2.py3-none-any.whl (194 kB)\nCollecting pytz==2017.2\n  Downloading pytz-2017.2-py2.py3-none-any.whl (484 kB)\nCollecting requests==2.18.3\n  Downloading requests-2.18.3-py2.py3-none-any.whl (88 kB)\nCollecting celery==4.1.0\n  Downloading celery-4.1.0-py2.py3-none-any.whl (400 kB)\nCollecting redis==2.10.6\n  Downloading redis-2.10.6-py2.py3-none-any.whl (64 kB)\nCollecting six>=1.5\n  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)\nCollecting urllib3<1.23,>=1.21.1\n  Downloading urllib3-1.22-py2.py3-none-any.whl (132 kB)\nCollecting idna<2.6,>=2.5\n  Downloading idna-2.5-py2.py3-none-any.whl (55 kB)\nCollecting chardet<3.1.0,>=3.0.2\n  Downloading chardet-3.0.4-py2.py3-none-any.whl (133 kB)\nCollecting certifi>=2017.4.17\n  Downloading certifi-2022.12.7-py3-none-any.whl (155 kB)\nCollecting kombu<5.0,>=4.0.2\n  Downloading kombu-4.6.11-py2.py3-none-any.whl (184 kB)\nCollecting billiard<3.6.0,>=3.5.0.2\n  Downloading billiard-3.5.0.5.tar.gz (150 kB)\nCollecting amqp<2.7,>=2.6.0\n  Downloading amqp-2.6.1-py2.py3-none-any.whl (48 kB)\nCollecting importlib-metadata>=0.18\n  Downloading importlib_metadata-4.8.3-py3-none-any.whl (17 kB)\nCollecting vine<5.0.0a1,>=1.1.3\n  Downloading vine-1.3.0-py2.py3-none-any.whl (14 kB)\nCollecting zipp>=0.5\n  Downloading zipp-3.6.0-py3-none-any.whl (5.3 kB)\nCollecting typing-extensions>=3.6.4\n  Downloading typing_extensions-4.1.1-py3-none-any.whl (26 kB)\nBuilding wheels for collected packages: backoff, billiard\n  Building wheel for backoff (setup.py): started\n  Building wheel for backoff (setup.py): finished with status 'done'\n  Created wheel for backoff: filename=backoff-1.4.3-py3-none-any.whl size=12171 sha256=208578fcfbfbbaa24e4ed938b3baf7ccd6adec7f20553ddfbe031fd4f85314d4\n  Stored in directory: /root/.cache/pip/wheels/b5/04/94/7b0bc7ef9c4cf45bcaeb72280f2461ebf91d96f24760bc0865\n  Building wheel for billiard (setup.py): started\n  Building wheel for billiard (setup.py): finished with status 'done'\n  Created wheel for billiard: filename=billiard-3.5.0.5-py3-none-any.whl size=87895 sha256=9c5a7c028812d98ed0dd0eea53112b2286a9b8131f66f530820c35ccb8b8b6bf\n  Stored in directory: /root/.cache/pip/wheels/ef/2b/96/4ccd58d823a9dac4828d9be5ab10d84eee79c2112662103099\nSuccessfully built backoff billiard\nInstalling collected packages: zipp, vine, typing-extensions, importlib-metadata, amqp, urllib3, six, pytz, kombu, idna, chardet, certifi, billiard, requests, redis, python-dateutil, psycopg2, gunicorn, Django, celery, backoff\nSuccessfully installed Django-1.11.4 amqp-2.6.1 backoff-1.4.3 billiard-3.5.0.5 celery-4.1.0 certifi-2022.12.7 chardet-3.0.4 gunicorn-19.7.1 idna-2.5 importlib-metadata-4.8.3 kombu-4.6.11 psycopg2-2.7.3 python-dateutil-2.6.1 pytz-2017.2 redis-2.10.6 requests-2.18.3 six-1.16.0 typing-extensions-4.1.1 urllib3-1.22 vine-1.3.0 zipp-3.6.0\n\u001b[91mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\n\u001b[0m\u001b[91mWARNING: You are using pip version 21.2.4; however, version 21.3.1 is available.\nYou should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.\n\u001b[0mRemoving intermediate container 570557883bf9\n ---> 3e4d1e9795ae\nStep 6/10 : ADD . /code/\n ---> dcb3495fbca7\nStep 7/10 : EXPOSE 8000\n ---> Running in b082b3cd045e\nRemoving intermediate container b082b3cd045e\n ---> f8ab7072bf60\nStep 8/10 : RUN python manage.py collectstatic -link --noinput\n ---> Running in 9db21925b89e\n\u001b[91mTraceback (most recent call last):\n  File \"/code/zenhub_charts/settings.py\", line 14, in <module>\n    from zenhub_charts.credentials import *\nModuleNotFoundError: No module named 'zenhub_charts.credentials'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"manage.py\", line 22, in <module>\n    execute_from_command_line(sys.argv)\n  File \"/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py\", line 363, in execute_from_command_line\n    utility.execute()\n  File \"/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py\", line 307, in execute\n    settings.INSTALLED_APPS\n  File \"/usr/local/lib/python3.6/site-packages/django/conf/__init__.py\", line 56, in __getattr__\n    self._setup(name)\n  File \"/usr/local/lib/python3.6/site-packages/django/conf/__init__.py\", line 41, in _setup\n    self._wrapped = Settings(settings_module)\n  File \"/usr/local/lib/python3.6/site-packages/django/conf/__init__.py\", line 110, in __init__\n    mod = importlib.import_module(self.SETTINGS_MODULE)\n  File \"/usr/local/lib/python3.6/importlib/__init__.py\", line 126, in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n  File \"<frozen importlib._bootstrap>\", line 994, in _gcd_import\n  File \"<frozen importlib._bootstrap>\", line 971, in _find_and_load\n  File \"<frozen importlib._bootstrap>\", line 955, in _find_and_load_unlocked\n  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n  File \"/code/zenhub_charts/settings.py\", line 16, in <module>\n    raise ImportError(\"Don't forget to create zenhub_charts/credentials.py\")\nImportError: Don't forget to create zenhub_charts/credentials.py\n\u001b[0mRemoving intermediate container 9db21925b89e\n","stderr":"The command '/bin/sh -c python manage.py collectstatic -link --noinput' returned a non-zero code: 1\n"},"endTime":1672670080417}